clients/privacy-center/public/fides-consent-demo.html

```diff
@@ -1,62 +1,75 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
```

Comment:
Apologies for the formatting changes, I didn't notice VSCode doing this at the time!